### PR TITLE
API authorization + User and Account entities ingestion

### PR DESCRIPTION
**Pull request by RPGPH**
Changelogs:
- Add API authorization and test
- Ingest User resource
- Ingest Account resource
- Establish User and Account relationship
- Create preliminary testing (using env variables)
- Temporary: Loaded environment variables to pre-push test (cc8510d)

Graph:
user-account

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "test:env": "LOAD_ENV=1 yarn test",
     "test:ci": "yarn lint && yarn type-check && yarn test",
     "build": "tsc -p tsconfig.dist.json --declaration",
-    "prepush": "yarn lint && yarn type-check && jest --changedSince master",
+    "prepush": "yarn lint && yarn type-check && LOAD_ENV=1 yarn jest --changedSince master",
```
Comment (eXtremeX) [inline package.json]: Clever :slightly_smiling_face:

File: src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+export const relationships = {
+  USER_HAS_ACCOUNT: {
+    _type: 'wp_engine_user_has_account',
+    _class: RelationshipClass.HAS,
+    sourceType: entities.USER._type,
+    targetType: entities.ACCOUNT._type,
+  },
+};
```
Comment (eXtremeX) [inline src/constants.ts]: Thanks for following the new pattern! You missed the `Record<>` type, I have included that in my PR (that's one of the changes I made).

File: src/steps/account.ts [hunk 3]
```diff
@@ -1,52 +1,62 @@
...
+      jobState.addEntity(accountEntity),
+      jobState.addRelationship(
+        createDirectRelationship({
+          _class: RelationshipClass.HAS,
+          from: userEntity,
+          to: accountEntity,
+        }),
+      ),
+    ]);
+  });
```
Comment (eXtremeX) [inline src/steps/account.ts]: I have left one comment about this in my PR, please check it out!

**Review by eXtremeX — COMMENTED** [review]
Looking good overall, appreciate the changelog and the graph image! :rocket: 

There were few small changes I wanted to suggest to you - Perhaps a bit unconventional, but I've made a PR that targets this one with those changes and I've included comments in it. https://github.com/creativice/graph-wpengine/pull/2

I'd suggest you take a look at it and let me know if you've got any questions about why I did something. If everything makes sense, we should accept that PR, and then you can continue with that structure (which you can/should use going forward as well). (*.snap and *. har are auto-generated files).

I've also added polly.js recordings (so `yarn test` and `LOAD_ENV=1 yarn test` both work now).

**Comment by RPGPH** [comment]
Hi Stefan! @eXtremeX 

I merged the recommended changes from PR #2 and cleaned up the code a little bit. For your review. :)

**Review by eXtremeX — APPROVED** [review]
Looks great @RPGPH! Please continue by ingesting additional resources and building up the relationships between them (frequent and smaller PRs would work great, but of course feel free to include few resources in any given PR depending on how connected they are).

**Pull request merged**